### DISCUSSION TITLE
SATB w/ piano

### DISCUSSION THREAD

**Discussion by visionm45** [BreezeWhite/oemer#45] [Q&A] [answered]
Hi. would oemer be able to handle sheet music with multiple parts on the same page? For example a choral piece with separate soprano alto tenor bass and piano parts.  It seems not to me because when I run it oemer produces a single musicxml part.

**Comment by BreezeWhite** [comment, marked as answer]
Yes and no. `oemer` is designed to automatically determine how many parts are there based on the ***distances between each stafflines***. If all distances between stafflines are equally distributed, then `oemer` would think there is only one part. On the other hand, if there are two distinct groups of distances between stafflines, then there probably contains more than one part.
Although `oemer` indeed has the ability to distinguish how many parts were there (to some extent), the decoding flow is only able to process two parts (e.g. two-handed piano sheet) at most. This is the reason for 'no' I mentioned at the beginning.

**Reply by visionm45** [reply]
okay that's what I was thinking I guess I would have to do quite a bit of editing then to get it to turn out the way I want because all I have is the combined SATB sheet music.  It did accurately identify every in the sheet music though except that the parts should be separate.
